firebase-firestore/src/commonMain/kotlin/dev/gitlive/firebase/firestore/Timestamp.kt

```diff
@@ -0,0 +1,8 @@
+package dev.gitlive.firebase.firestore
+
+expect class Timestamp
+
+expect fun timestampNow(): Timestamp
```

Comment:
So there is no way to expect static (Companion) function in Kotlin?

Reply:
I don't think is supported yet